docs/source/routing/observability/telemetry/instrumentation/standard-instruments.mdx

```diff
@@ -116,20 +116,10 @@ The initial call to Uplink during router startup is not reflected in metrics.
 ### Telemetry
 - `apollo.router.telemetry.batch_processor.errors` - The number of errors encountered by exporter batch processors.
   - `name`: One of `apollo-tracing`, `datadog-tracing`, `jaeger-collector`, `otlp-tracing`, `zipkin-tracing`.
-  - `error` = One of `channel closed`, `channel full`.
-
-### Deprecated
```

Comment:
We should keep this section in 1.x and put the "Internals" header above it

Reply:
Oops [7e21f0e](https://github.com/apollographql/router/pull/6985/commits/7e21f0ed8115945e2ecca3c1ec4ee4d80eccf5c6)